assets/docs/sayakpaul/collections/vision_transformer/1.md

```diff
@@ -32,18 +32,19 @@ as reproducible as possible.
 
 The table below provides a performance summary:
 
-| **Model** | **Top-1 Accuracy (%)** |
-|:---:|:---:|
-| B/8 | 85.948 |
-| L/16 | 85.716 |
-| B/16 | 84.018 |
-| R50-L/32 | 83.784 |
-| R26-S/32 (light aug) | 80.944 |
-| R26-S/32 (medium aug) | 80.462 |
-| S/16 | 80.462 |
-| B/32 | 79.436 |
-
-Note that the top-1 accuracy is reported on ImageNet-1k validation set.
+| **Model** | **Top-1 Accuracy** | **Checkpoint** | **Misc** |
+|:---:|:---:|:---:|:---:|
```

Comment:
Ah now I see - these values are actually copied from the AugReg Colab dataframe in
https://colab.sandbox.google.com/github/google-research/vision_transformer/blob/master/vit_jax_augreg.ipynb
as verified by

```python
t='''| B/8 | 85.948 | B_8-i21k-300ep-lr_0.001-aug_medium2-wd_0.1-do_0.0-sd_0.0--imagenet2012-steps_20k-lr_0.01-res_224.npz |  |
| L/16 | 85.716 | L_16-i21k-300ep-lr_0.001-aug_medium1-wd_0.1-do_0.1-sd_0.1--imagenet2012-steps_20k-lr_0.01-res_224.npz |  |
| B/16 | 84.018 | B_16-i21k-300ep-lr_0.001-aug_medium2-wd_0.03-do_0.0-sd_0.0--imagenet2012-steps_20k-lr_0.03-res_224.npz |  |
| R50-L/32 | 83.784 | R50_L_32-i21k-300ep-lr_0.001-aug_medium1-wd_0.1-do_0.1-sd_0.1--imagenet2012-steps_20k-lr_0.01-res_224.npz |  |
| R26-S/32 (light aug) | 80.944 | R26_S_32-i21k-300ep-lr_0.001-aug_light0-wd_0.03-do_0.1-sd_0.1--imagenet2012-steps_20k-lr_0.03-res_224.npz | [tb.dev run](https://tensorboard.dev/experiment/8rjW26CoRJWdAR3ejtgvHQ/) |
| R26-S/32 (medium aug) | 80.462 | R26_S_32-i21k-300ep-lr_0.001-aug_medium2-wd_0.1-do_0.0-sd_0.0--imagenet2012-steps_20k-lr_0.01-res_224.npz |  |
| S/16 | 80.462 | S_16-i21k-300ep-lr_0.001-aug_light1-wd_0.03-do_0.0-sd_0.0--imagenet2012-steps_20k-lr_0.03-res_224.npz | [tb.dev run](https://tensorboard.dev/experiment/52LkVYfnQDykgyDHmWjzBA/) |
| B/32 | 79.436 | B_32-i21k-300ep-lr_0.001-aug_medium1-wd_0.03-do_0.0-sd_0.0--imagenet2012-steps_20k-lr_0.03-res_224.npz |  |'''
adapt_filenames = [l.split(' | ')[2][:-4] for l in t.split('\n')]
print(pd.concat([
    df.query(f'adapt_filename=="{adapt_filename}"')[['adapt_filename', 'adapt_final_test']]
    for adapt_filename in adapt_filenames
]).to_markdown(index=False))
```

which outputs

```
| adapt_filename                                                                                        |   adapt_final_test |
|:------------------------------------------------------------------------------------------------------|-------------------:|
| B_8-i21k-300ep-lr_0.001-aug_medium2-wd_0.1-do_0.0-sd_0.0--imagenet2012-steps_20k-lr_0.01-res_224      |            0.85948 |
| L_16-i21k-300ep-lr_0.001-aug_medium1-wd_0.1-do_0.1-sd_0.1--imagenet2012-steps_20k-lr_0.01-res_224     |            0.85716 |
| B_16-i21k-300ep-lr_0.001-aug_medium2-wd_0.03-do_0.0-sd_0.0--imagenet2012-steps_20k-lr_0.03-res_224    |            0.84018 |
| R50_L_32-i21k-300ep-lr_0.001-aug_medium1-wd_0.1-do_0.1-sd_0.1--imagenet2012-steps_20k-lr_0.01-res_224 |            0.83784 |
| R26_S_32-i21k-300ep-lr_0.001-aug_light0-wd_0.03-do_0.1-sd_0.1--imagenet2012-steps_20k-lr_0.03-res_224 |            0.80944 |
| R26_S_32-i21k-300ep-lr_0.001-aug_medium2-wd_0.1-do_0.0-sd_0.0--imagenet2012-steps_20k-lr_0.01-res_224 |            0.80462 |
| S_16-i21k-300ep-lr_0.001-aug_light1-wd_0.03-do_0.0-sd_0.0--imagenet2012-steps_20k-lr_0.03-res_224     |            0.80462 |
| B_32-i21k-300ep-lr_0.001-aug_medium1-wd_0.03-do_0.0-sd_0.0--imagenet2012-steps_20k-lr_0.03-res_224    |            0.79436 |
```

Could you mention this explicitly? Otherwise readers would think that you re-run the evaluation with the exported hub modules.

As mentioned in the email exchange, it would be great to add results of

- validation of one (or multiple) models on imagenet and report those numbers
- fine-tuning of one (or multiple) models; if you can't fine-tune on imagenet, it's fine to do it on some smaller dataset

You could for example leave the table you have, mention clearly that the one row is copied from the Colab so it's clear where those numbers come from, and add additional columns with your results from the TF hub modules, including links to the Colab / code that was used to produce exactly those numbers.